src/lib/services/gauging-station-service.js

```diff
@@ -23,6 +23,15 @@ const getGaugingStationLicencesById = async gaugingStationId => {
   throw new NotFoundError(`Could not find gauging station conditions with ID ${gaugingStationId}`);
 };
 
+const getGaugingStationsByLicenceId = async licenceId => {
+  const gaugingStations = await gaugingStationRepo.findGaugingStationByLicenceId(licenceId);
+  if (gaugingStations) {
+    return gaugingStations;
+  }
+  throw new NotFoundError(`Could not find gauging station conditions with ID ${licenceId}`);
```

Comment:
I'm wondering if this error handling is not necessary:

If a licence has no connected gauging stations, I would expect the returned payload to be an empty array `[]`, whereas this would return an error. 

Please consider changing this to:

```
const getGaugingStationsByLicenceId = async licenceId => gaugingStationRepo.findGaugingStationByLicenceId(licenceId);```